site/app/templates/grading/electronic/Status.twig

```diff
@@ -606,16 +606,17 @@
                                 <b>Students who have submitted:</b>  {{ submitted_total }} / {{ total_submissions }} ({{ submitted_percentage }}%)
                                 <br/>
                                 <br/>
-                                <b>Current percentage of grading done:</b> {{ graded_total }} / {{ submitted_total }} ({{ graded_percentage }}%)
+                                <b>Current percentage of TA grading done:</b> {{ graded_total }} / {{ submitted_total }} ({{ graded_percentage }}%)
                             {% endif %}
                         
                             <br/>
                             <br/>
                         {% endif %}
+                        {# To be used when functionality to view peer data in instructor view is to be added 
                         {% if peer %}
-                            <b>Current percentage of student grading done:</b> {{ peer_percentage }}% ({{ peer_graded }}/{{ peer_total }})
-                        {% else %}
-                            <b>By Grading Sections:</b>
+                        <b>Current percentage of student grading done:</b> {{ peer_percentage }}% ({{ peer_graded }}/{{ peer_total }})
```

Comment:
Can this commented out code be removed?

Reply:
I was thinking we might need this once we set up the function for peer view access from non student views. That is this will be in use later. Although, if you want, i can get it removed.

Reply:
I updated the work on Peer Grading Percentage from Peer View. I earlier got confused with variable naming. Now, it takes number of students peer has to grade too. 
P.S: Although a better approach would have been to calculate percentage as [Graded Students / Students who submitted] differing from the current case [Graded Students / Total Peers assigned to the grader], but it looks like there is too much  transitivity on a variable $submitted_total which is supposed to represent number of submissions, in both peer and non peer views. I tried digging deeper but for no good. Although it works perfectly fine in case of TA view , it does not seem to work well with peer view. I was hoping to get this corrected when we devise a new paradigm on accessing peer view from non peer identities.